lib/instream/deleter/predicate.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Instream.Deleter.Predicate do
+  @moduledoc """
+  Point deleter for the line protocol.
+
+  ### InfluxDB v2.x Options
+
+  - `bucket`: delete data to a specific bucket
+  - `org`: delete data to a specific organization
+  """
+
+  alias Instream.Query.Headers
+  alias Instream.Query.URL
+
+  @behaviour Instream.Deleter
+
+  @impl Instream.Deleter
+  def delete(points, opts, conn) do
+    config = conn.config()
+    headers = Headers.assemble(config, opts) ++ [{"Content-Type", "application/json"}]
+    body = Jason.encode!(points)
```

Comment:
`Jason` should everywhere be replaced with a suitable call to `Instream.Connection.JSON` to keep the option of replacing the JSON library.

Reply:
Yeah, it was so obvious, but I missed it. Thank you, fixed.